core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -1,21 +1,36 @@
 <script type="text/ng-template" id="components/socialButtons">
+<style type="text/css">
+  
+i.oppia-share-btn-fa {
+    display: inline-block;
```

Comment:
Undoing my comment... 
@seanlip  
I alphabetized the properties. Kindly look whether they are according to oppia styling standards.